daemons/attrd/attrd_utils.c

```diff
@@ -232,7 +232,6 @@ attrd_free_attribute_value(gpointer data)
     free(v->nodename);
     free(v->current);
     free(v->requested);
-    free(v->node_xml_id);
```

Comment:
We can drop it from the `attribute_value_t` struct too

Reply:
Oops, meant to